run-tests-in-docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+#if [ ${USE_UID:-0} -ne 0 ] && [ ${USE_GID:-0} -ne 0 ]; then
+#  groupadd -f -g $USE_UID abs
+#  useradd -g $USE_GID -M -l -u $USE_UID abs -d $ABS_DIR -s /bin/bash || true
+#fi
+
+#chown -R $USE_UID:$USE_GID $ABS_DIR
+#sudo --preserve-env=PYTHONPATH,PATH -g "#$USE_GID" -u "#$USE_UID" -- python -m app_build_suite $@
+#pipenv run pre-commit run --all-files
```

Comment:
Is this file required at all? You could at least remove the commented out lines

Reply:
Yes, it's WIP - will clean it up :)